variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "name" {
+  description = "Name of the repository"
+  default     = ""
```

Comment:
Let's remove the default values for `name`, `namespace`, and `stage`

Reply:
Removed